apps/admin-portal/src/components/email-templates/add-template-type.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+import React, { ReactElement, FunctionComponent } from "react";
+import { Forms, Field } from "@wso2is/forms";
+import { Grid, GridRow, GridColumn } from "semantic-ui-react";
+
+/**
+ * Interface to capture role basics props.
+ */
+interface AddEmailTemplateTypeProps {
```

Comment:
Check other places as well.

```suggestion
interface AddEmailTemplateTypePropsInterface {
```